code/game/machinery/computer/shuttle_computers.dm

```diff
@@ -426,6 +455,20 @@
 		to_chat(usr, "<span class='info'>You insert \the [SC] into \the [src].</span>")
 		src.updateUsrDialog()
 
+/obj/machinery/computer/shuttle_control/proc/use_pass(obj/item/weapon/card/shuttle_pass/P, mob/user)
+	if(!istype(P))
+		return
+
+	if(user.drop_item(P, src))
+		if(shuttle && shuttle.type == P.allowed_shuttle)
+			if(shuttle.travel_to(P.destination, src, user))
```

Comment:
If the shuttle doesn't go, it should get to the put_in_hands down below to give the card back